dataprofiler/profilers/numerical_column_stats.py

```diff
@@ -1453,7 +1460,7 @@ def _fold_histogram(
         return [bin_counts_pos, bin_edges_pos], [bin_counts_neg, bin_edges_neg]
 
     @property
-    def median_abs_deviation(self) -> float:
+    def median_abs_deviation(self) -> Union[float, np.float64]:
```

Comment:
is the return type correct here given the possible `return` values in the function (i.e. `return np.nan`, `return 0` (int),  etc..)?

Reply:
We could just do `return 0.0` if we wanted. But I think for that, we should be fine.

Reply:
true `0.0` should cover it. `np.nan` is `float64`

Reply:
@Sanketh7

Reply:
Changed it to `return 0.0`